doespy/doespy/util.py

```diff
@@ -279,6 +279,26 @@ def json_query(data, expr):
     env.filters["ceil"] = ceil
     env.filters["floor"] = floor
 
+    # load design specific fiters
+    filter_folder = get_suite_design_dir() + "/filter_plugins"
+    source_files = [file for file in os.listdir(filter_folder) if file.endswith(".py")]
+    for file in source_files:
+        try: 
+            path = filter_folder + '/' + file
+            DesignFilterSpec = importlib.util.spec_from_file_location('design_filter_plugins', path)
+            DesignFilterModule = importlib.util.module_from_spec(DesignFilterSpec)
+            DesignFilterSpec.loader.exec_module(DesignFilterModule)
+            design_filters = DesignFilterModule.FilterModule().filters()
+        except:
+            continue
+        if isinstance(design_filters, dict):
+            for name, function in design_filters.items():
+                env.filters[name] = function
```

Comment:
Should we allow filters with the same name from different files or raise an error?